templates/aws/aws_cloudformation_template.yml

```diff
@@ -5,16 +5,28 @@ Description:
   CloudFormation template for LANSA Database regression Testing.
 
 Parameters:
-  LANSA_VERSION:
+  LANSAVERSION:
     Type: String
     Description: LANSA Version that needs to be test
-  AMI_ID:
+  AMIID:
     Type: String
-    Description: AMI_ID for provisioning the VM    
-  MYSQL_SNAPSHOT_ARN:
+    Description: AMI_ID for provisioning the VM
+  SECURITYGROUPID:
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: DB_REGRESSION_TESTING_VM_SG_ID
```

Comment:
How is this going to work when there are multiple stacks?

Reply:
This is a Security Group Id which is fetched from parameter store. So it is possible to attach same SG to multiple VM's